py4web/utils/form.py

```diff
@@ -455,7 +455,8 @@ def __init__(
                 record_id = self.record and self.record.get("id")
                 if not post_vars.get("_delete"):
                     validated_vars = {}
-                    for field in self.table:
+                    for fn in [k for k in post_vars.keys() if not k in ['_formname', '_formkey']]:
+                        field =  self.table[fn]                
```

Comment:
we should do the opposite.
```
for field in self.table:
    if not field.name in post_vars: continue
```
else field =  self.table[fn]    may result in an unhandled error or a vulnerability.

Reply:
Done. Thank. you

Reply:
I don't' know the workflow to fix a PR I hope that is the right way.